pkg/azuredisk/controllerserver.go

```diff
@@ -890,9 +891,11 @@ func (d *Driver) CreateSnapshot(ctx context.Context, req *csi.CreateSnapshotRequ
 			}
 		case consts.ResourceGroupField:
 			resourceGroup = v
+		case consts.LocationField:
```

Comment:
Need to add support for the "location" parameter.

Reply:
Replicated to `CloudProvisioner`